Stripe/PublicHeaders/STPPaymentMethodParams.h

```diff
@@ -184,6 +190,17 @@ NS_ASSUME_NONNULL_BEGIN
                                         billingDetails:(STPPaymentMethodBillingDetails *)billingDetails
                                               metadata:(nullable NSDictionary<NSString *, NSString *> *)metadata;
 
+/**
+ Creates params for a giropay PaymentMethod;
+
+ @param eps   An object containing additional EPS details.
+ @param billingDetails  An object containing the user's billing details. Note that `billingDetails.name` is required for giropay PaymentMethods.
+ @param metadata     Additional information to attach to the PaymentMethod.
+ */
++ (nullable STPPaymentMethodParams *)paramsWithEPS:(STPPaymentMethodEPSParams *)eps
```

Comment:
How come this is `nullable`?

Reply:
most of the other methods here are nullable. I think they should be since the depend on a [[self alloc] init]?

Reply:
Huh I didn't think about that.

> The init method defined in the NSObject class does no initialization; it simply returns self. In terms of nullability, callers can assume that the NSObject implementation of init does not return nil.

I think this means we can safely mark them as nonnull unless we add logic to return nil.  This is good because it improves the Swift ergonomics!

Reply:
change to `nonnull` as suggested